app/controllers/mfa_confirmation_controller.rb

```diff
@@ -22,6 +22,7 @@ def skip
 
   def new
     session[:password_attempts] ||= 0
+    user_session[:context] = 'reauthentication'
```

Comment:
I mention the above because this controller will redirect to TOTP or Phone without the user getting to choose, and it may not work for accounts that don't have those MFAs: https://github.com/18F/identity-idp/blob/727925506eb1cbdd0d0a790d5b010a390c8b464e/app/controllers/mfa_confirmation_controller.rb#L47-L51

Reply:
Do we need feature spec coverage for the changed behavior?

Reply:
My hunch is we want to avoid `user_session[:context] = 'reauthentication'` here since it is hard to see the impact elsewhere (even if the tests pass). For the most part, I'd like to move away from "session context" except where it's absolutely needed (like phone OTP verification).